chat-client-ui-types/src/uiContracts.ts

```diff
@@ -1,5 +1,5 @@
-import { InsertToCursorPositionParams } from '@aws/language-server-runtimes-types'
-export { InsertToCursorPositionParams } from '@aws/language-server-runtimes-types'
+import { InsertToCursorPositionParams, QuickActionsOptions } from '@aws/language-server-runtimes-types'
+export { InsertToCursorPositionParams, QuickActionsOptions } from '@aws/language-server-runtimes-types'
```

Comment:
Why does client/destination need to know about these options? Is Toolkit expected to manipulate the options somehow instead of just forwarding the initialize response?

Reply:
I agree with Viktor that passing the whole InitializeResult would be too wasteful for what we are trying to achieve. I can create a more generic type like `defaultTabDataOptions` that only contains quick actions for the time being and that can be extended in the future, but I don't think initializeResult is the right approach

Reply:
`defaultTabData` prefix as a public-facing interface is a bit confusing outside of chat client codebase imo. Something more concrete would be better, like `chatOptions` or `chatConfiguration`?

Reply:
I'm fine with the suggested approach.
But the original question is still in place: why does this options type need to be exposed to client/destination?